src/app/equipment/equipment.cpp

```diff
@@ -0,0 +1,12 @@
+#include "equipment.hpp"
+
+Equipment::Equipment(const std::string &name, const double price, const int amount, const std::string &material,
+                     const std::string &description)
+    : Product(name, price, amount), material_(material), description_(description)
+{
+}
+
+std::vector<std::pair<std::string, std::string>> Equipment::getInfo() const
```

Comment:
Because we use pointers to Product, those getters must be placed as virtual functions in Product class and also in Medicine class and it leads to creation of unnecessary code in those classes.

Reply:
Adjustment to std::map

Reply:
Done